### ISSUE TITLE
Missing ARM based CPU support

### ISSUE THREAD

**Issue by JanMikes** [dockette/adminer#37]
Hi, on ARM based cpus (M1 macbook for example) containers fails to start, due to incompatible architecture:
```
adminer_1  | standard_init_linux.go:219: exec user process caused: exec format error
```

There is easy solution for that when commands are under your control:

```
docker buildx create --use
docker buildx build --push --platform linux/arm/v7,linux/arm64/v8,linux/amd64 --tag your-username/multiarch-example:gitlab .
```

But as long as it seems there are automated builds, i am unsure how it could be done here, i just found this resource that might be helpful:  https://github.com/ckulka/docker-multi-arch-example

**Comment by JanMikes** [comment]
Closed via #38

**Comment by f3l1x** [comment]
Excellent work 👍
